pos_customer_display/static/src/js/customer_display.js

```diff
@@ -203,7 +203,10 @@ odoo.define('pos_customer_display', function(require) {
             var res = OrderlineSuper.prototype.set_quantity.call(this, quantity);
             if (quantity != 'remove') {
                 var line = this;
-                this.pos.prepare_text_customer_display('add_update_line', {'line': line});
+                if(this.selected)
```

Comment:
Please adhere to the style from the surrounding code: keep curly opening bracket on the same line, and apply indents of length 4.